src/controllers/user.controller.ts

```diff
@@ -0,0 +1,85 @@
+import {Request, Response} from 'express';
+import db from '../database/sequelize';
+import {getAutoSuggestUsers} from '../services/user.service';
+const User = db.user;
+
+class UserController {
+  static readonly DEFAULT_USERS_LIMIT = 10;
+  static readonly LOGIN_SUBSTRING = '';
+
+  async getUsers(req: Request, res: Response): Promise<any> {
+    User.findAll({raw: true})
+    .then(users => {
+      const filteredUsers = getAutoSuggestUsers(users as any[], UserController.LOGIN_SUBSTRING, UserController.DEFAULT_USERS_LIMIT);

```

Comment:
For users it is Model<any, any>[] type. I was thinking how to change it to IUser[] for getAutoSuggestUsers function, but without success. I will try to solve it

Reply:
Could you check https://sequelize.org/master/manual/typescript.html and https://dev.to/jctaveras/sequelize-typescript-what-you-need-to-know-41mj out, maybe it would help

Reply:
Now I use - users: IUser[]